content/blog/scheduling-in-react/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Scheduling in React"
+date: "2019-03-15T12:00:00.000Z"
+---
+
+In modern applications, user interfaces often have to juggle multiple tasks at the same time: A search component needs to respond to user input while providing auto completion results; An interactive dashboard needs to update the charts while loading data from the server and sending analytics data to your backend.
```

Comment:
These are examples of the juggling?

"A search component needs to respond to user input while providing auto completion results; An interactive dashboard needs to update the charts while loading data from the server and sending analytics data to your backend."

Reply:
The "it happens very often" is awkward. I would shorten that even to

"In modern applications, we often have to juggle multiple tasks at the same time"

WDYT? Does that still say what you want it to say? Or are you saying the applications themselves are responsible for multiple tasks at the same time?

Reply:
This works as well. Thank you! 😊